commandapi-kotlin-annotations/src/test/kotlin/dev/jorel/commandapi/ktannotations/AnnotationsTest.kt

```diff
@@ -0,0 +1,34 @@
+package dev.jorel.commandapi.ktannotations
+
+import dev.jorel.commandapi.ktannotations.tests.*
+import dev.jorel.commandapi.ktannotations.tests.Testable
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.DynamicTest.dynamicTest
+import org.junit.jupiter.api.TestFactory
+import java.io.File
+import kotlin.test.assertTrue
+
+internal val kspDir =
+	"target/generated-sources/ksp-java/${Testable::class.java.packageName.replace('.', '/')}"
```

Comment:
This is a little arbitrary, yes. But I don't know any other solution myself, so guess this is it.